server/server.py

```diff
@@ -2,26 +2,34 @@
 import socketserver
 import socket
 
-#set the process name to "chaos_server" so we can easily kill it with "pkill chaos_server"
+# set the process name to "chaos_server" so we can easily kill it with
+# "pkill chaos_server"
```

Comment:
Instead of moving this further away from the function it's documenting, it should probably be made into a docstring.